bindings/python/iota_sdk/client/client.py

```diff
@@ -425,8 +425,8 @@ def build_and_post_block(self,
 
         options = {k: v for k, v in options.items() if v != None}
 
-        if 'output' in options:
-            options['output'] = options.pop('output').as_dict()
+        if 'params' in options:
+            options['output'] = options.pop('params').as_dict()
```

Comment:
`options['output']` ? Is it output in the message interface ?

Reply:
Yes https://github.com/iotaledger/iota-sdk/blob/85942111ac19a3998e945e2810563178178dab49/sdk/src/client/api/block_builder/mod.rs#L75
and this type is actually not the same as SendParams

Reply:
🤔

Reply:
yeah, I wondered whether I should change that type as well.

Reply:
The other fields wouldn't be considered unless you also change that which I don't think we should do

Reply:
Let's ignore for now, added an item to https://github.com/iotaledger/iota-sdk/issues/129